src/components/expanded-state/chart/chart-data-labels/ChartPercentChangeLabel.js

```diff
@@ -25,7 +25,24 @@ const PercentLabel = styled(AnimatedTextInput)`
   ${android && `margin-vertical: -19px;`}
 `;
 
-export default function ChartPercentChangeLabel() {
+function formatNumber(num) {
```

Comment:
I feel like there's already a bignumber.js format option for this, but just leaving a note here and we can take a look later